test/dummy/db/schema.rb

```diff
@@ -989,6 +988,7 @@
     t.string   "banner_2_texto_alt"
     t.boolean  "enviada",              default: false
     t.string   "dominio_de_envio"
+    t.string   "suscribible_type"
```

Comment:
Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.